src/test/java/ibm/jceplus/junit/base/BaseTestXDHKeyImport.java

```diff
@@ -125,18 +125,18 @@ public void createKeyPairXDHParamImport(String alg) throws Exception {
                     "515095759487624245475052955143775821008860566299846242602604658885545774489");
             scalar = Base64.getDecoder().decode("RtcJeC6VA0w1s2ly58eimwgmjNnIvdBQ0WwXodcOKx0=");
             actualPbk = Base64.getDecoder()
-                    .decode("MCowBQYDK2VuAyEAmU2lHYDsKSa7YoPkUeOxazej9sWb9m2hYbHEl6uIIwE=");
+                    .decode("MCwwBwYDK2VuBQADIQCZTaUdgOwpJrtig+RR47FrN6P2xZv2baFhscSXq4gjAQ==");
```

Comment:
Should we change all versions to this new value?

Reply:
I updated the Javadoc associated with this change to document what the difference is. I will also plan on making a very similar update to other releases if this looks ok to you.

Reply:
Makes sense to me.